doc/api/vm.md

```diff
@@ -167,10 +167,17 @@ const contextifiedSandbox = vm.createContext({ secret: 42 });
     in stack traces produced by this `Module`.
   * `columnOffset` {integer} Specifies the column number offset that is
     displayed in stack traces produced by this `Module`.
-  * `initalizeImportMeta` {Function} Called during evaluation of this `Module`
+  * `initializeImportMeta` {Function} Called during evaluation of this `Module`
     to initialize the `import.meta`. This function has the signature `(meta,
     module)`, where `meta` is the `import.meta` object in the `Module`, and
     `module` is this `vm.SourceTextModule` object.
+  * `importModuleDynamically` {Function} Called during evaluation of this
```

Comment:
Perhaps call this `resolveDynamicImport`?

Reply:
Or `loadDynamicImport` if resolve is misleading. The adverb form just feels a bit clumsy for some reason here.

Reply:
I agree the name is a bit over the top but I haven't been able to come up with anything better

Reply:
How about just `dynamicImport`?

Reply:
I'd point out that the name is inherited from the spec proposal, and thus consistent with the origin of the name `initializeImportMeta` (which also came from the spec proposal).

Reply:
Good point, sure.